src/main/webapp/app/editor/instructions/editor-instructions.component.ts

```diff
@@ -419,7 +424,12 @@ export class EditorInstructionsComponent implements AfterViewInit, OnChanges, On
             if (label === this.translateService.instant('arTeMiSApp.editor.testStatusLabels.noResult')) {
                 text += '<span class="text-danger bold">' + label + '</span>'; // this should be bold
             } else {
-                text += '<a data-tests="' + tests.toString() + '" class="test-status"><span class="text-danger result">' + label + '</span></a>';
+                text +=
+                    '<a data-tests="' +
```

Comment:
Can you please keep this statement in one line?

Reply:
Done; this change was being added automatically on commit by some Git hook.